### ISSUE TITLE
custom yaml policy to support yaml/json scan framework

### ISSUE THREAD

**Issue by junhu73** [bridgecrewio/checkov#6629]
**Describe the feature**

Does checkov support yaml custom policy to scan yaml or json document framework?

**Examples**

given yaml or json document, if the certain key or value are missing in the document as defined in custom yaml policy, scan will succeed or fail.

**Additional context**

**Comment by mannycepeda1989** [comment]
Good afternoon @junhu73 please see the supported Checkov frameworks.

https://www.checkov.io/3.Custom%20Policies/YAML%20Custom%20Policies.html#:~:text=Policies%20in%20code-,Supported%20Frameworks,-Ansible

JSON or YAML shouldn't be an issue as Cloudformation templates commonly use either language to describe what AWS resources you want to create and configure.

YAML policies are also supported as mentioned in the Docs:
https://www.checkov.io/3.Custom%20Policies/YAML%20Custom%20Policies.html

**Comment by gruebel** [comment]
no, but you should be able to achieve this with a simple Python check.
